packages/app/src/cli/prompts/webhook/options-prompt.ts

```diff
@@ -47,13 +48,33 @@ export async function optionsPrompt(flags: WebhookTriggerFlags): Promise<Webhook
     address: '',
   }
 
+  const availableVersions = await requestApiVersions()
```

Comment:
I'd recommend not doing API interactions from `prompts` and instead pass that any information necessary for the prompt from the outside:

```js
async function optionsPrompt({flags, availableApiVersions}: OptionsPromptOptions)
```

Then the command service would do the following:

```js
async function service({flags}) {
  const session = await ensureAuthenticatedPartners()
  const apiVersions = await requestApiVersions()
  const options = optionsPrompt({flags, apiVersions})
  // ...
}
```
Note that the command service becomes the place where all the blocks are combined as a sequence of steps that's easy to follow. By placing `ensureAuthenticatedPartners` we prevent the scattering of `ensure...` throughout the services.